releases/development/master/extra/install-chef-suse.sh

```diff
@@ -572,6 +586,11 @@ check_repo_tag key     11.3 SUSE-Cloud-5-Updates
 check_repo_tag summary 11.3 SLE11-HAE-SP3-Pool     'SUSE Linux Enterprise High Availability Extension 11 SP3' 'false'
 check_repo_tag repo    11.3 SLE11-HAE-SP3-Updates  'updates://zypp-patches.suse.de/autobuild/SLE_HAE/11-SP3/update/x86_64' 'false'
```

Comment:
Well, `$REPOS_SKIP_CHECK` is really intended as a temporary solution until the repos become available. So it should generally not be abused.

In this case, since you just don't care about SLES11, skipping the validation tests just seems like the natural thing to do to me. But maybe I'm missing something?